README.md

```diff
@@ -10,80 +10,21 @@ This repository contains the manifest files for the **Windows Package Manager**
 > [!IMPORTANT]
 > At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.
 
-The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).
+The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring private repositories offering private WinGet package hosting, see [private repositories](doc/private/README.md).
 
-# Submitting a Package
+## Repology
+[Repology](https://repology.org/), the packaging hub monitors package repositories comparing package versions and other information.
 
-To submit a package to this repository, you should follow these steps:
+[![Repository status](https://repology.org/badge/repository-big/winget.svg)](https://repology.org/repository/winget)
 
-1. Follow the [Contributing](#contributing) guidelines below.
-2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
-3. [Test](#test-your-manifest) your manifest.
-4. [Submit](#submit-your-pr) your pull request (PR).
-5. Respond to any feedback in your PR.
+# Documentation
```

Comment:
Can this be moved above Repology? I get the feeling new users are more interested in the documentation